recipes/cuda-cudart/meta.yaml

```diff
@@ -0,0 +1,290 @@
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("cuda-cudart-static_" + target_platform, exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcudart_static.a  # [linux]
+        - test -L $PREFIX/lib/libcudadevrt.a      # [linux]
+        - test -L $PREFIX/lib/libculibos.a        # [linux]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Runtime native Libraries
+      description: |
+        CUDA Runtime native Libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-cudart-static_{{ target_platform }}
+    build:
+      noarch: generic
+    files:
+      - targets/{{ target_name }}/lib/*.a  # [linux]
+      - Library\lib\cudart_static.lib      # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcudart_static.a  # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcudadevrt.a      # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libculibos.a        # [linux]
+        - if not exist %LIBRARY_LIB%\cudart_static.lib exit 1               # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Runtime native Libraries
+      description: |
+        CUDA Runtime native Libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-driver-dev
+    build:
+      skip: True  # [not linux]
+    files:
+      - lib/stubs/libcuda.so
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - cuda-driver-dev_{{ target_platform }}
```

Comment:
Should this be versioned in some kind of way? Otherwise if someone asks for a specific version of `cuda-driver-dev` they could get a newer or older version of `cuda-driver-dev_linux-64` for example.